### PR TITLE
attempt to fix -e in actions

### PR DESCRIPTION
**Pull request by kylecarow**

### CODE REVIEW

**Comment by calbaker** [comment]
@kylecarow, what's the deal with this PR?

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -41,5 +41,5 @@ jobs:
       # https://github.nrel.gov/MBAP/fastsim/issues/310
       - name: Python unit tests 
         run: |
-          pip install -e ".[dev]" && pytest -v python/fastsim/tests/
```
Comment (calbaker) [inline .github/workflows/tests.yaml:44]: @kylecarow looks like maybe we should undo these changes in a new commit, change the PR description from `attempt to fix -e in actions` to `failed attempt to fix -e in actions with other minor patch changes`

**Comment by kylecarow** [comment]
> @kylecarow, what's the deal with this PR?

This explains it https://github.nrel.gov/MBAP/fastsim/issues/310

Installing the package will go faster if it is non-editable
